### ISSUE TITLE
Strange bug : textColor not set when called from a tabris serve app

### ISSUE THREAD

**Issue by natanraj** [eclipsesource/tabris-js#1384]
### Problem description

```js
new tabris.TextView(
{
        layoutData:{centerX:0, top:20},
        textColor:'rgba(250, 1, 1, 1)',
        text:'rgba(250, 1, 1,1)'
}
).appendTo(tabris.ui.contentView);
```
Expecting the font to be in Red color when served from my dev server (thru tabris serve); but shown in black color (default)
### Expected behavior

Red color text is shown. 

### Environment

- Tabris.js version: 
- Device: Samsung
- OS:  android

**But scratchpad is showing correctly**

### Code snippet

```javascript
new tabris.TextView(
{
        layoutData:{centerX:0, top:20},
        textColor: 'rgba(250, 1, 1, 1)',
        text:'rgba(250, 1, 1,1)'
}
).appendTo(tabris.ui.contentView);


```

**Comment by mpost** [comment]
I can not confirm your obersvation. Works as expected:

![image](https://user-images.githubusercontent.com/73655/27226218-36388474-529e-11e7-8cdc-56054985667f.png)

**Comment by natanraj** [comment]
Will try in one more device and see. That is why i too called as strange

**Comment by ralfstx** [comment]
@natanraj Please reopen when you have more information.

**Comment by 3to1null** [comment]
I got the same problem, following code is not producing colored text in Tabris 2 Dev app:
```
new TextView({
            top: 15, left: 1, right: 0,
            text: 'test',
            font: '18px',
            textColor: "#ff0000"
        }).appendTo(ui.contentView)
```

Using the scratchpad it is displaying colored text.

I tried on a Honor 8 (android 7), Huawei Ascend P7 (slightly modded android 4.4), LG G3s (android 5). I'm using the latest version of the android dev app.

**Comment by ralfstx** [comment]
For me the snippet shows red text in the latest Developer App. Tested with `tabris@nightly`. @3to1null Which tabris version do you have?

**Comment by mpost** [comment]
@3to1null I have tried the exact snippet you posted above (https://github.com/eclipsesource/tabris-js/issues/1384#issuecomment-309867006) on several different devices with the tabris 2.0 nightly. They all show a red color. Please make sure you have updated your tabris.js module.

**Comment by 3to1null** [comment]
I'm sorry, I thought the Tabris-CLI by default installed the latest version of Tabris.js. I fixed it by updating to 2.0.0-rc2.

Sorry for bothering you, this issue can be closed again.

**Comment by mpost** [comment]
Thanks for the heads up. As you know by now the tabris version is controlled your declaration in the package.json.
